Editor/EdgegapWindowV2.cs

```diff
@@ -812,6 +826,8 @@ private async Task verifyApiTokenGetRegistryCreds()
             if (!_isApiTokenVerified)
             {
                 UpdateUI();
+                _isVerifyingToken = false;
```

Comment:
I think we need to reset this boolean even if verification fails, otherwise you can't rerun verification and you're stuck. 
please can you move this statement above the if statement `if (!_isApiTokenVerified)` before we update the UI?